src/info/manifest.c

```diff
@@ -1712,8 +1728,9 @@ manifestBuildIncr(Manifest *this, const Manifest *manifestPrior, BackupType type
 
                     // Preserve values if the file will not be copied, is possibly equal to the prior file, or will be stored with
```

Comment:
This comment looks like it needs updating?

Reply:
The comment is still true, but I added a comment to clarify why in 02f5415.